vernac/g_vernac.mlg

```diff
@@ -337,12 +334,12 @@ GRAMMAR EXTEND Gram
   ] ]
   ;
   finite_token:
-    [ [ IDENT "Inductive" -> { (Inductive_kw,Finite) }
-      | IDENT "CoInductive" -> { (CoInductive,CoFinite) }
-      | IDENT "Variant" -> { (Variant,BiFinite) }
-      | IDENT "Record" -> { (Record,BiFinite) }
-      | IDENT "Structure" -> { (Structure,BiFinite) }
-      | IDENT "Class" -> { (Class true,BiFinite) } ] ]
+    [ [ IDENT "Inductive" -> { (Inductive_kw) }
```

Comment:
You can remove the parentheses around `Inductive_kw` (and similarly for the rules below), it will make it look less like LISP ;)